_posts/2019-09-24-rest-api-with-strapi-io.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "Designing RESTful APIs with Strapi"
+categories: [ Software development for founders ]
+tags: [software development, founder, startup]
```

Comment:
I propose to remove the 'founder' tag and add the 'strapi' tag. How do you think?